From f3f6c7e4aa62cf7075fcb4ffd5ce7662e56a7b99 Mon Sep 17 00:00:00 2001
From: Travis Newman <travis.c.newman@gmail.com>
Date: Sun, 26 Jan 2014 10:59:39 -0500
Subject: [PATCH 1/2] Added alt text to image_spec.rb

---
 spec/zemus/image_spec.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/spec/zemus/image_spec.rb b/spec/zemus/image_spec.rb
index 6c43c5e..05c2bba 100644
--- a/spec/zemus/image_spec.rb
+++ b/spec/zemus/image_spec.rb
@@ -4,7 +4,7 @@ describe Zemus::Image do
   it "embeds images" do
     url = "http://google.com/image.gif"
     Zemus::Image.new(url).to_embed.
-      should eq("<img src='http://google.com/image.gif' class='img-responsive img-thumbnail' />")
+      should eq("<img src='http://google.com/image.gif' class='img-responsive img-thumbnail' alt='http://google.com/image.gif' />")
   end
 
-end
\ No newline at end of file
+end
-- 
1.8.3.4 (Apple Git-47)


From ce6885f08c5f58006de9af3f3b82709c2f5af65a Mon Sep 17 00:00:00 2001
From: Travis Newman <travis.c.newman@gmail.com>
Date: Sun, 26 Jan 2014 11:00:34 -0500
Subject: [PATCH 2/2] Added alt text to image.rb

---
 lib/zemus/image.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/zemus/image.rb b/lib/zemus/image.rb
index ce799c0..cfa9899 100644
--- a/lib/zemus/image.rb
+++ b/lib/zemus/image.rb
@@ -11,8 +11,8 @@ module Zemus
     end
 
     def to_embed
-      "<img src='#{@url}' class='img-responsive img-thumbnail' />"
+      "<img src='#{@url}' class='img-responsive img-thumbnail' alt='#{@url}' />"
     end
 
   end
-end
\ No newline at end of file
+end
-- 
1.8.3.4 (Apple Git-47)

